runner/runner.go

```diff
@@ -334,9 +421,23 @@ func (r *Runner) DispatchWorkflowJob(hookTargetType, signature string, jobData [
 
 	switch HookTargetType(hookTargetType) {
 	case RepoHook:
-		poolManager, err = r.findRepoPoolManager(job.Repository.Owner.Login, job.Repository.Name)
```

Comment:
This feels more verbose than needed. Couldn't ```findRepoPoolManager()``` be refactored to use ```GetRepoPoolManager()```? You don't need to remove ```findRepoPoolManager()``` to be able to test.

Reply:
`GetRepoPoolManager()` replaces `findRepoPoolManager()`. 
It is only the function signature that differs, which requires an extra step to get the repo from store.
The `r.store.GetRepository(r.ctx, owner, name)` was previously part of `findRepoPoolManager()`.

I could keep `findRepoPoolManager()` on the runner if you want. But I wanted to refactor away all the pool manager functions for the runner.

Reply:
Sounds great. I will add again `findRepoPoolManager` and `findOrgPoolManager`.